### PR TITLE
Add spectre_v1 kernel parameter testcase

### PR DESCRIPTION
**Pull request by wjn740**
- Verification run: http://mitigations.qa2.suse.asia/tests/6659

### CODE REVIEW

**Comment by wjn740** [comment]
@badboywj @skliu @simonlm @tonyyuan1

**Comment by wjn740** [comment]
@alice-suse

File: tests/cpu_bugs/spectre_v1.pm [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# without any warranty.
+
+# Summary: CPU BUGS on Linux kernel check
+# Maintainer: James Wang <jnwang@suse.com>
+
+package spectre_v1;
+use strict;
+use warnings;
+use base "consoletest";
+use base "Mitigation";
```
Comment (alice-suse) [inline tests/cpu_bugs/spectre_v1.pm]: Generally it is not recommended to use multiple inherit. What's the base class of Mitigation? Is it feasible to make inherit order like baseclass -> consoletest -> Mitigation -> spectre_v1?

Comment (wjn740) [reply tests/cpu_bugs/spectre_v1.pm]: I could remove 'consoletest'

**Comment by alice-suse** [comment]
LGTM

File: lib/main_common.pm [hunk 2]
```diff
@@ -2429,6 +2429,9 @@ sub load_mitigation_tests {
     if (get_var('KVM_GUEST_MIGRATION')) {
         loadtest "cpu_bugs/kvm_guest_migration";
     }
+    if (get_var('SPECTRE_V1')) {
```
Comment (wjn740) [inline lib/main_common.pm]: @Walterkd

**Pull request merged**